pennylane_lightning/core/src/simulators/lightning_gpu/utils/cuStateVecError.hpp

```diff
@@ -0,0 +1,110 @@
+// Copyright 2022-2024 Xanadu Quantum Technologies Inc.
```

Comment:
New files shouldn't have a starting year. right?

```suggestion
// Copyright 2024 Xanadu Quantum Technologies Inc.
```